### PR TITLE
Add Task1

### PR DESCRIPTION
**Pull request by patwys**

### CODE REVIEW

File: learning/basics/task1/PW-input_print.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name = str(input("Hi there. What's your name? "))
```
Comment (PKurowski93) [inline learning/basics/task1/PW-input_print.py:1]: There is no need to type str() command beacuse result of input is always a string

**Review by PKurowski93 — APPROVED** [review]
You should add void lines between  3-5 lines acc. to pep-8.  Logic of code is correct :).

**Review by PKurowski93 — APPROVED** [review]
Well done. 🙂

**Review by Artur-Ws — APPROVED** [review]
Always remember to add exactly one empty lina at the end of file according to PEP-8, generally it looks fine :)

**Pull request merged**